### PR TITLE
echo exercise

### PR DESCRIPTION
**Pull request by tsjoberg**

### CODE REVIEW

File: echo.js [hunk 1]
```diff
@@ -1,5 +1,7 @@
 function echo(){
-  // ...
+  for (i=0;  i < arguments.length; i++){
```
Comment (afeld) [inline echo.js]: You have one problem with this line... what is it?

Comment (tsjoberg) [inline echo.js]: is it a functional or syntax issue? The only thing I can see right now is there is an extra space between the first semicolon and i < arguments.length (just committed to remove space)?

**Comment by tsjoberg** [comment]
Read up on efficient techniques on for loops and added arg.length into a variable (so it only calculates once) and added var for i to properly scope the variable (believe this was the original error you first commented on).

File: echo.js (hunk 1 above)
Comment (tsjoberg) [inline echo.js]: Please check newest commit - removed global "i" var which I believe is the problem you have identified

**Comment by afeld** [comment]
Awesome, thanks for the fixes!

**Pull request closed without merging**